app/controllers/appointments_controller.rb

```diff
@@ -59,7 +59,7 @@ def search_params # rubocop:disable Metrics/MethodLength
   def notify_customer(appointment)
     return unless appointment.notify?
 
-    Appointments.customer(appointment, booking_location.object).deliver_later
+    AppointmentChangeNotificationJob.perform_later(appointment)
```

Comment:
Cool, will do. I believe the features are already testing that the mail's sent, so should just need to add something for the activity.